### ISSUE TITLE
Have lts.conf-assigned computer names shown even when client is off.

### ISSUE THREAD

**Issue by dgroos** [epoptes/epoptes#38]
There is a half-way step to the convenience of use request described in #6. That great suggestion is for client icons to keep their place in the View window even when they are turned off. Another option (and not mutually exclusive) is having the client icons maintain the names assigned to them in the lts.conf even when turned off. For example, considering a machine labelled such in lts.conf:
[aa:bb:cc:dd:ee:ff]
Computer1
when a computer is turned off it would be labelled with the MAC address "aa:bb:cc:dd:ee:ff". However,  it would be much more useful to show "Computer1". This allows me to see instantly which computers are turned off (don't have to do process of elimination). Also, it allows me to turn on select computers depending on my instructional or tech-work needs.

**Comment by alkisg** [comment]
Upvoting this. I think that ltsp.conf HOSTNAME parameters should show up in a special "ltsp.conf" read-only group in the Epoptes UI.

**Comment by alkisg** [comment]
Rethinking about this, it would probably be better for ltsp.conf to have the following form.
This will allow a single ltsp.conf to contain multiple groups, separated with comment lines in the form `# EPOPTES_GROUP=group-name`:

```ini
# EPOPTES_GROUP=class-a
[mac-of-a01]
HOSTNAME=a01

[mac-of-a01]
HOSTNAME=a02
...

# EPOPTES_GROUP=class-b
[mac-of-b01]
HOSTNAME=b01

[mac-of-b02]
HOSTNAME=b02
...
```

**Comment by alkisg** [comment]
Closed by GSoC 2023 Epoptes improvements #204.
Please test if the [proposed PPA build](https://epoptes.org/documentation/ppa/) resolves this issue.
